python/cvmfs/repository.py

```diff
@@ -292,10 +459,9 @@ def retrieve_certificate(self):
         return Certificate(certificate)
 
 
-    @abc.abstractmethod
     def retrieve_file(self, file_name):
-        """ Abstract method to retrieve a file from the repository """
-        pass
+        """ Method to retrieve a file from the repository """
```

Comment:
Or wait... you mean the user should only use the retrieve_object method to get files? Because that could be... retrieve_file gives total flexibility to fetch whatever you want, including metadata if necessary, but retrieve_object can only do it from the 'data' folder.
On a different matter, if retrieve_object is actually going to retrieve files, it should be called retrieve_file, isn't it?